CI: Python test_examples.py failing due to httpx.ReadTimeout

## Problem

Python CI tests are failing intermittently due to `httpx.ReadTimeout` errors.

```
httpx.ReadTimeout: The read operation timed out
FAILED tests/test_examples.py::test_example[examples/openai_bot_rag.py]
============= 1 failed, 12 passed, 1 skipped in 178.16s (0:02:58) ==============
```

## Analysis

The test harness has skip logic for transient external service errors, but `httpx.ReadTimeout` (`"The read operation timed out"`) doesn't match the existing patterns (`"Timeout"`, `"Request timed out"`).

## Proposed Fix

Add `"read operation timed out"` to the transient error skip list in `python-sdk/tests/test_examples.py`.

--- a/python-sdk/tests/test_examples.py
+++ b/python-sdk/tests/test_examples.py
@@ -197,6 +197,7 @@ async def test_example(example_file: str):
                         "Connection error",
                         "Timeout",
                         "Request timed out",
+                        "read operation timed out",
                     ]
                 ):
                     pytest.skip(

